docs/modules/ROOT/pages/commandline_args.adoc

```diff
@@ -0,0 +1,20 @@
+=== --csi-endpoint
+
+*Required*: true
+
+*Multiple values*: false
+
+*Environment variable:* `CSI_ENDPOINT`
+
+The path to the https://github.com/container-storage-interface/spec/blob/master/spec.md[Container Storage Interface] Unix Domain Socket
+that the operator should listen on.
+
+=== --node-name
```

Comment:
The `node` subcommand needs both parameters but the `controller` subcommand only needs `--csi-endpoint`. It could make sense to describe the subcommands in this page. But it could also make sense to delete this page because you cannot run the operator with `cargo run` locally and the user never gets in touch with these parameters.

Reply:
Deleted in 828ac4cb90f6da8169fea0a177b08b9c12570589